src/theory/theory_inference_manager.h

```diff
@@ -105,6 +105,11 @@ class TheoryInferenceManager
    * been provided in a custom way.
    */
   void trustedConflict(TrustNode tconf);
+  /** Send lemma lem with property p on the output channel. */
+  LemmaStatus lemma(TNode lem, LemmaProperty p = LemmaProperty::NONE);
+  /** Send (trusted) lemma lem with property p on the output channel. */
+  LemmaStatus trustedLemma(TrustNode tlem,
+                           LemmaProperty p = LemmaProperty::NONE);
```

Comment:
I'd suggest to pass stuff by `const&`...
```suggestion
  LemmaStatus trustedLemma(const TrustNode& tlem,
                           LemmaProperty p = LemmaProperty::NONE);
```